biolink-model.yaml

```diff
@@ -6145,6 +6145,717 @@ slots:
+      For details see: Bouma, G. (2009). Normalized (pointwise) mutual information
+      in collocation extraction. Proceedings of GSCL, 30, 31-40.
+      https://svn.spraakdata.gu.se/repos/gerlof/pub/www/Docs/npmi-pfd.pdf
+    range: string
+
+  normalized pointwise mutual information max:
+    is_a: association slot
+    description: >-
+      Normalized Pointwise Mutual Information Max is a variant of NPMI
+      proposed by Bouma 2009 whereby the PMI is normalized by -ln(max(p(x),p(y))).
+      Using this normalization factor will result in an asymmetric measure whereby
+      NPMImax=1 when one concept is occurs only in the context of another, but not
+      necessarily the other way around. For details see: Bouma, G. (2009). Normalized
+      (pointwise) mutual information in collocation extraction. Proceedings of GSCL,
+      30, 31-40. https://svn.spraakdata.gu.se/repos/gerlof/pub/www/Docs/npmi-pfd.pdf
+    range: string
+
+  mutual dependence:
+    is_a: association slot
+    description: >-
+      Mutual Dependency (MD), often also known as PMI\xB2, is a variant
+      of PMI that attempts to overcome PMI's preference for rare events. For details,
+      see Thanopoulos, A., Fakotakis, N., & Kokkinakis, G. (2002, May). Comparative
+      Evaluation of Collocation Extraction Metrics. In LREC (Vol. 2, pp. 620-625).
+      http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.11.8101&rep=rep1&type=pdf
+    range: string
+
+  log frequency biased mutual dependence:
+    is_a: association slot
+    description: >-
+      Log-Frequency-Biased Mutual Dependency is a variant of Mutual Dependency
+      that results in favoring the more frequently occuring pairs when ranking. For
+      details, see Thanopoulos, A., Fakotakis, N., & Kokkinakis, G. (2002, May). Comparative
+      Evaluation of Collocation Extraction Metrics. In LREC (Vol. 2, pp. 620-625).
+      http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.11.8101&rep=rep1&type=pdf
+    range: string
+
+  tmkp concept1 count:
+    is_a: association slot
+    description: The raw count of observations of one member (concept1) of a cooccurring
+      pair of concepts in a text corpus.
+    range: string
+
+  tmkp concept2 count:
+    is_a: association slot
+    description: The raw count of observations of one member (concept2) of a cooccurring
+      pair of concepts in a text corpus.
+    range: string
+
+  tmkp concept cooccurrence count:
+    is_a: association slot
+    description: The raw count of observations of two concepts cooccurring in the
+      same document (or part of a document) in a text corpus.
+    range: string
+
+  contribution towards patient survival:
+    is_a: association slot
+    description: >-
+      Contribution is a summary statistic indicating the level of contribution
+      an entity has on patient survival time as measured by its quantifiable impact
+      on a distribution of patient survival time.
+    range: integer
```

Comment:
I think this is a typo and you mean float

But the first example is a range and not a float...

Reply:
We may also want to define our own types here. Can also indicate bounds, e.g. between 0 and 1